### PR TITLE
Prem HTML B1,B2,B3,B4

### PR DESCRIPTION
**Pull request by PremRS**
Status Update

### CODE REVIEW

File: prem/FloatHolyGrail.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+            <span>Sidebar 1</span>
+        </div>
+        <div class="article">
+            <span> Article</span>
+        </div>
+        <div class="sidebar2">
+            <span>Sidebar 2</span>
+        </div>
+    </section>
+    <div class="foot">
```
Comment (iammadhan) [inline prem/FloatHolyGrail.html]: change this to footer

File: prem/FloatHolyGrail.html [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+        .sidebar2 {
+            height: 100%;
+            float: left;
+            text-align: center;
+        }
+    </style>
+</head>
+
+<body>
+    <div class="head">
```
Comment (iammadhan) [inline prem/FloatHolyGrail.html]: change this to header

File: prem/DropDownNavbar.html [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        }
+    </style>
+</head>
+
+<body>
+    <div class="navigation">
+        <a href="#">Home</a>
+        <a href="#">About</a>
+        <div class="drop-menu">
+            <button class="drop-button">Services </button>
```
Comment (iammadhan) [inline prem/DropDownNavbar.html]: Why this is a button? change it to a <a>

File: prem/DropDownNavbar.html [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+            padding: 20px;
+            display: inline-block;
+        }
+
+        a {
+            text-decoration: none;
+            color: #ffffff;
+        }
+
+        .navigation>a {
```
Comment (iammadhan) [inline prem/DropDownNavbar.html]: add space before and after >

File: prem/DropDownNavbar.html [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+            padding: 10px 10px;
+        }
+
+        .menu>a {
```
Comment (iammadhan) [inline prem/DropDownNavbar.html]: add space before and after >

File: prem/FloatHolyGrail.html (hunk 1 above)
Comment (PremRS) [reply prem/FloatHolyGrail.html]: Sure, i vl change it and push it

File: prem/FloatHolyGrail.html (hunk 2 above)
Comment (PremRS) [reply prem/FloatHolyGrail.html]: I will change it and push it

File: prem/DropDownNavbar.html (hunk 3 above)
Comment (PremRS) [reply prem/DropDownNavbar.html]: I will change it to <a> and push it

File: prem/DropDownNavbar.html (hunk 5 above)
Comment (PremRS) [reply prem/DropDownNavbar.html]: Ok , i vl correct it and push the code

File: prem/DropDownNavbar.html (hunk 4 above)
Comment (PremRS) [reply prem/DropDownNavbar.html]: Ok, I will change it

**Pull request merged**